recipes/hmmlearn/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hmmlearn" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 694646f8302bc6402925a4b6892f3a5ccede06d25f22157c18cfbdecdb748361
+
+build:
+  number: 0
+  skip: True  # [py37]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
```

Comment:
hmmlearn has compiled code, you should indicate this using:
```
build:
    - {{ compiler('c') }}
```
If hmmlearn requires a C++ compiler as well add `- {{ compiler('cxx') }}`